website/src/pages/home/_UseCases.tsx

```diff
@@ -0,0 +1,105 @@
+        link: '/use-cases/onboarding',
+    },
+    {
+        icon: CheckboxMarkedCircleOutlineIcon,
+        title: 'Resolve incidents',
+        description: 'Identify the root cause in code and fix the issue everywhere, faster.',
+        // link: '/use-cases/incidents'
+    },
+    {
+        icon: CachedIcon,
+        title: 'Promote code reuse',
+        description: 'Find existing code for reuse and contribute to a more coherent codebase.',
+        // link: '/use-cases/code-reuse'
+    },
+    {
+        icon: LaptopIcon,
+        title: 'Boost code health',
+        description: 'Improve code health with large-scale changes, and track key initiatives.',
+        // link: '/use-cases/code-health'
+    },
+]
+
+const UseCases: FunctionComponent = () => (
+    <div className="position-relative">
+        <ContentSection className="py-7">
+            <div className="text-center">
+                <h1 className="font-weight-bold">Move fast &mdash; even in big codebases</h1>
+                <a href="https://docs.sourcegraph.com">See the docs</a>
+            </div>
+
+            <div className="row max-w-800 mx-auto mt-6">
+                {useCases.map((useCase, i) => (
+                    <div
+                        key={useCase.title}
+                        className={classNames(
+                            {
+                                ['mb-6']: i !== useCases.length - 1,
+                                ['mb-0']: i === useCases.length - 1,
+                            },
+                            'col-sm-6 d-flex flex-column flex-sm-row align-items-center align-items-sm-start'
+                        )}
+                    >
+                        <useCase.icon className="text-vivid-violet w-100 max-w-50 h-auto mr-sm-2 mb-4 mb-sm-0" />
+                        <div className="text-center text-sm-left">
+                            <h4 className="font-weight-bold">{useCase.title}</h4>
+                            <div className="text-lg">
+                                <p className="m-0">{useCase.description}</p>
+                                {useCase.link && <Link to={useCase.link}>Learn more</Link>}
+                            </div>
+                        </div>
+                    </div>
+                ))}
+            </div>
+        </ContentSection>
+
+        <div
+            className={classNames(
+                styles.floatingBox,
+                'col-6 bg-gradient-venus px-4 py-7 p-sm-7 mx-auto mx-lg-0 lg-absolute right-0 bottom-0 max-w-550'
+            )}
+        >
+            <h3 className="font-weight-bold mb-4">Want to use Sourcegraph at your company?</h3>
+            <p className="text-xl">
+                <Link to="/get-started">Get started</Link> for free with up to 10 teammates or{' '}
+                <Link to="/demo">request a demo</Link> to learn about our enterprise plan and to see Sourcegraph in your
+                own environment.
+            </p>
+        </div>
```

Comment:
Thanks @zlonko! Yeah, the floating behaviour beyond our desktop artboard size was intentional but this is good to note for our stakeholder feedback since we don't have beyond-desktop artboards to follow.

I think what we can do here is move the box inside the container if we want to preserve it being within the constraint when the viewport width has reached the intended artboard size.

Reply:
Fixed this with a more responsive solution in [this commit](https://github.com/sourcegraph/about/pull/5276/commits/1970c6cde4ad5ad2d4b23a8ba10f1902ae595263)